core/services/vrf/delegate_test.go

```diff
@@ -242,7 +239,12 @@ func TestStartingCounts(t *testing.T) {
 			EncodedPayload: []byte{},
 		},
 	}
-	require.NoError(t, db.Create(&txes).Error)
+	sql := `INSERT INTO eth_txes (nonce, from_address, to_address, encoded_payload, value, gas_limit, state, created_at, broadcast_at, meta, subject, evm_chain_id, min_confirmations, pipeline_task_run_id, simulate) 
+			VALUES (:nonce, :from_address, :to_address, :encoded_payload, :value, :gas_limit, :state, :created_at, :broadcast_at, :meta, :subject, :evm_chain_id, :min_confirmations, :pipeline_task_run_id, :simulate);`
+	for _, tx := range txes {
+		_, err = db.NamedExec(sql, &tx)
```

Comment:
Doesn't matter that much since this is in a test, but it's possible to insert all the txes in a single DB query by passing an array:

https://github.com/smartcontractkit/chainlink/blob/45be2432dbc60dd55cf75bd5a11398c9e641effc/core/services/pipeline/orm.go#L268-L271

Reply:
I tried this and this did not work for some reasons, that's why I made a loop. However, I will make another attempt in my next "gorm removal PR"

Reply:
Just checked this again and here is the error:
```
panic: reflect: call of github.com/jmoiron/sqlx/reflectx.(*Mapper).TraversalsByNameFunc on slice Value [recovered]
        panic: reflect: call of github.com/jmoiron/sqlx/reflectx.(*Mapper).TraversalsByNameFunc on slice Value
```
but with the loop it works.